bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -431,3 +431,63 @@ spec:
         operator: "Exists"
       - key: "node.kubernetes.io/network-unavailable"
         operator: "Exists"
+---
+
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-master-metrics
+  namespace: openshift-ovn-kubernetes
+  annotations:
```

Comment:
We should probably add a `networkoperator.openshift.io/non-critical: ""` annotation here.